server/src/test/java/org/elasticsearch/discovery/ClusterDisruptionIT.java

```diff
@@ -156,7 +157,7 @@ public void testAckedIndexing() throws Exception {
                                 int shard = Math.floorMod(Murmur3HashFunction.hash(id), numPrimaries);
                                 logger.trace("[{}] indexing id [{}] through node [{}] targeting shard [{}]", name, id, node, shard);
                                 IndexRequestBuilder indexRequestBuilder = client.prepareIndex("test", "type", id)
-                                    .setSource("{}", XContentType.JSON)
+                                    .setSource(Map.of("f" + randomIntBetween(1, 20), randomNonNegativeLong()), XContentType.JSON)
```

Comment:
Let's randomly choose the max number of dynamic fields to test at the beginning of the test (randomized between 1 and 10) and then randomly select a field between those here